assets/src/components/Sidebar/CampaignSidebar.js

```diff
@@ -112,7 +126,9 @@ export class CampaignSidebar extends Component {
   }
 
   render() {
-    const { parent, theme } = this.state;
+    const { parent, theme, meta } = this.state;
+
+    const isLegacyTheme = !theme || isLegacy(theme.id);
```

Comment:
Correct me if I'm wrong but this validation is also added inside the `isLegacy` function [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/589/files#diff-030a0990cba4c830359767f7c640b7d3e58da55f7926014d6c1d4bf97c1d6d28R20)

Reply:
I first also thought this after seeing your comment, but it's actually slightly different :grin: In this case `theme` is an object that is loaded from a JSON file, and not the name.